llvm/test/tools/llvm-split/AMDGPU/large-kernels-merging.ll

```diff
@@ -15,19 +15,20 @@
 ; Also check w/o large kernels processing to verify they are indeed handled
 ; differently.
 
-; P0 is empty
```

Comment:
I'm not claiming this is the correct fix for the test, but it gets it passing. I haven't updated the comments so those are now stale. @Pierre-vh what would you expect this test to check, with a fixed ratio?